datadog_lambda/tracing.py

```diff
@@ -699,6 +699,8 @@ def inject_correlation_ids():
 
 
 def set_dd_trace_py_root(trace_context_source, merge_xray_traces):
+    if not _is_context_complete(dd_trace_context):
```

Comment:
also would it make sense to push this check further up the stack? or does it need to be here because we know that we're going to be messing with the global trace context and therefore we need to make sure we're going to be safe?

Reply:
Good question.

The reason it is here and not above is because this method is called from a different file. Moving the check up the stack would mean checking the global in the other file... bad news.

Reply:
fair enough